### ISSUE TITLE
`edge trace-observer list` seems to be broken in v0.14.0

### ISSUE THREAD

**Issue by sanderblue** [newrelic/newrelic-cli#404]
### Description

Per @jthurman42 

> Edge tracing doesn't seem to working in v0.14.0
    ```$ newrelic edge trace-observer list -a 1
FATAL[2020-10-12T13:24:38-07:00] Cannot query field "traceObservers" on type "EdgeTraceObserver". Did you mean "traceFilters"?, null, Cannot query field "errors" on type "EdgeTraceObserver"., null```

This looks like a potential upstream API change affecting this.

**Comment by jthurman42** [comment]
Trace level output:
```
$ newrelic edge trace-observer list -a ${NEW_RELIC_ACCOUNT_ID}
DEBUG[2020-10-12T13:28:35-07:00]/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/newrelic/newrelic-client-go@v0.42.1/internal/logging/structured_logger.go:72 github.com/newrelic/newrelic-client-go/internal/logging.StructuredLogger.Debug() performing request                            method=POST url="https://api.newrelic.com/graphql"
TRACE[2020-10-12T13:28:35-07:00]/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/newrelic/newrelic-client-go@v0.42.1/internal/logging/structured_logger.go:77 github.com/newrelic/newrelic-client-go/internal/logging.StructuredLogger.Trace() request details                               headers="{\"Api-Key\":[\"************************\"],\"Content-Type\":[\"application/json\"],\"Newrelic-Requesting-Services\":[\"newrelic-cli|newrelic-client-go\"],\"User-Agent\":[\"newrelic-cli/0.14.0 (https://github.com/newrelic/newrelic-cli)\"]}" query="query($accountId: Int!) { actor { account(id: $accountId) { edge { tracing { traceObservers { traceObservers { status providerRegion name id endpoints { https { url port host } endpointType agent { port host } status } } errors { type message } } } } } } }" variables="{\"accountId\":****************}"
TRACE[2020-10-12T13:28:41-07:00]/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/newrelic/newrelic-client-go@v0.42.1/internal/logging/structured_logger.go:77 github.com/newrelic/newrelic-client-go/internal/logging.StructuredLogger.Trace() request completed                             body="{\"errors\":[{\"locations\":[{\"column\":4,\"line\":2}],\"message\":\"Cannot query field \\\"traceObservers\\\" on type \\\"EdgeTraceObserver\\\". Did you mean \\\"traceFilters\\\"?\"},{\"locations\":[{\"column\":3,\"line\":21}],\"message\":\"Cannot query field \\\"errors\\\" on type \\\"EdgeTraceObserver\\\".\"}]}" headers="{\"Cache-Control\":[\"max-age=0, private, must-revalidate\"],\"Content-Security-Policy\":[\"frame-ancestors *.newrelic.com\"],\"Content-Type\":[\"application/json; charset=utf-8\"],\"Date\":[\"Mon, 12 Oct 2020 20:28:40 GMT\"],\"Proxied-By\":[\"Service Gateway\"],\"Served-By\":[\"nerd-graph\"],\"Server\":[\"Cowboy\"],\"Strict-Transport-Security\":[\"max-age=31536000; includeSubDomains\"]}" method=POST status_code=200 url="https://api.newrelic.com/graphql"
FATAL[2020-10-12T13:28:41-07:00]/private/tmp/newrelic-cli-20201002-31889-12w02lk/newrelic-cli-0.14.0/internal/utils/utils.go:54 github.com/newrelic/newrelic-cli/internal/utils.LogIfFatal() Cannot query field "traceObservers" on type "EdgeTraceObserver". Did you mean "traceFilters"?, null, Cannot query field "errors" on type "EdgeTraceObserver"., null
```

**Comment by ctrombley** [comment]
Resolved by #393
